### PR TITLE
Run Prettier on HTML code fences, part 14

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #20638

The PR focuses only on HTML code fences.

Idea is to gradually prettify all the CSS code fences before the full automation.

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/app_structure/index.md [hunk 1]
```diff
@@ -74,36 +74,55 @@ From the HTML point of view, the app shell is everything outside the content sec
...
+    <link rel="manifest" href="js13kpwa.webmanifest" />
+    <script src="data/games.js" defer></script>
+    <script src="app.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <p>
+        <a class="logo" href="https://js13kgames.com"
+          ><img src="img/js13kgames.png" alt="js13kGames"
+        /></a>
```
Comment (OnkarRuikar) [inline files/en-us/web/progressive_web_apps/app_structure/index.md:100]:
```suggestion
        <a class="logo" href="https://js13kgames.com">
        		<img src="img/js13kgames.png" alt="js13kGames" />
        </a>
```

File: files/en-us/web/progressive_web_apps/app_structure/index.md [hunk 2]
```diff
@@ -74,36 +74,55 @@ From the HTML point of view, the app shell is everything outside the content sec
...
+    <main>
+      <h1>js13kGames A-Frame entries</h1>
+      <p class="description">
+        List of games submitted to the
+        <a href="https://js13kgames.com/aframe">A-Frame category</a> in the
+        <a href="https://2017.js13kgames.com">js13kGames 2017</a> competition.
+        You can
+        <a href="https://github.com/mdn/pwa-examples/blob/master/js13kpwa"
+          >fork js13kPWA on GitHub</a
+        >
```
Comment (OnkarRuikar) [inline files/en-us/web/progressive_web_apps/app_structure/index.md:112]:
```suggestion
        You can
        <a href="https://github.com/mdn/pwa-examples/blob/master/js13kpwa">fork js13kPWA on GitHub</a>
```

File: files/en-us/web/progressive_web_apps/structural_overview/index.md [hunk 3]
```diff
@@ -92,40 +92,53 @@ The HTML in the file `index.html` creates the structure of the app. The parts th
...
+    <link rel="manifest" href="js13kpwa.webmanifest" />
+    <script src="data/games.js" defer></script>
+    <script src="app.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <p>
+        <a class="logo" href="http://js13kgames.com"
+          ><img src="img/js13kgames.png" alt="js13kGames"
+        /></a>
```
Comment (OnkarRuikar) [inline files/en-us/web/progressive_web_apps/structural_overview/index.md:116]:
```suggestion
        <a class="logo" href="http://js13kgames.com">
          <img src="img/js13kgames.png" alt="js13kGames" />
        </a>
```

File: files/en-us/web/progressive_web_apps/structural_overview/index.md [hunk 4]
```diff
@@ -92,40 +92,53 @@ The HTML in the file `index.html` creates the structure of the app. The parts th
...
+    <main>
+      <h1>js13kGames A-Frame entries</h1>
+      <p class="description">
+        List of games submitted to the
+        <a href="http://js13kgames.com/aframe"> A-Frame category</a> in the
+        <a href="http://2017.js13kgames.com">js13kGames 2017</a> competition.
+        You can
+        <a href="https://github.com/mdn/pwa-examples/blob/master/js13kpwa">
+          fork js13kPWA on GitHub</a
+        >
```
Comment (OnkarRuikar) [inline files/en-us/web/progressive_web_apps/structural_overview/index.md:128]:
```suggestion
        <a href="https://github.com/mdn/pwa-examples/blob/master/js13kpwa">
          fork js13kPWA on GitHub</a>
```

File: files/en-us/web/svg/applying_svg_effects_to_html_content/index.md [hunk 5]
```diff
@@ -156,10 +160,11 @@ You could also apply a color matrix:
 ```html
 <svg height="0">
   <filter id="f2">
-    <feColorMatrix values="0.3333 0.3333 0.3333 0 0
+    <feColorMatrix
+      values="0.3333 0.3333 0.3333 0 0
                            0.3333 0.3333 0.3333 0 0
```
Comment (OnkarRuikar) [inline files/en-us/web/svg/applying_svg_effects_to_html_content/index.md:165]:
```suggestion
              0.3333 0.3333 0.3333 0 0
```

File: files/en-us/web/svg/attribute/order/index.md [hunk 6]
```diff
@@ -70,9 +75,10 @@ svg {
 </table>
 
 - `<number-optional-number>`
+
```
Comment (OnkarRuikar) [inline files/en-us/web/svg/attribute/order/index.md:78]:
```suggestion
```

File: files/en-us/web/svg/attribute/order/index.md [hunk 7]
```diff
@@ -70,9 +75,10 @@ svg {
...
   - : This value indicates the number of cells in each dimension for the kernel matrix. The values provided must be {{cssxref("integer")}}s greater than zero. Values that are not integers will be truncated, i.e. rounded to the closest integer value towards zero. The first number, indicates the number of columns in the matrix. The second number, indicates the number of rows in the matrix. If no second number is not provided, it defaults to the first number.
 
-      It is recommended that only small values (e.g., 3) be used; higher values may result in very high CPU overhead and usually do not produce results that justify the impact on performance.
+    It is recommended that only small values (e.g., 3) be used; higher values may result in very high CPU overhead and usually do not produce results that justify the impact on performance.
```
Comment (OnkarRuikar) [inline files/en-us/web/svg/attribute/order/index.md:81]:
```suggestion
      It is recommended that only small values (e.g., 3) be used; higher values may result in very high CPU overhead and usually do not produce results that justify the impact on performance.
```

**Review by OnkarRuikar — COMMENTED** [review]
Correct split tags

File: files/en-us/web/svg/applying_svg_effects_to_html_content/index.md [hunk 8]
```diff
@@ -156,10 +160,11 @@ You could also apply a color matrix:
 ```html
 <svg height="0">
   <filter id="f2">
-    <feColorMatrix values="0.3333 0.3333 0.3333 0 0
+    <feColorMatrix
+      values="0.3333 0.3333 0.3333 0 0
+              0.3333 0.3333 0.3333 0 0
                            0.3333 0.3333 0.3333 0 0
```
Comment (OnkarRuikar) [inline files/en-us/web/svg/applying_svg_effects_to_html_content/index.md:166]:
```suggestion
              0.3333 0.3333 0.3333 0 0
```

File: files/en-us/web/svg/applying_svg_effects_to_html_content/index.md [hunk 9]
```diff
@@ -156,10 +160,11 @@ You could also apply a color matrix:
...
-                           0.3333 0.3333 0.3333 0 0
-                           0      0      0      1 0"/>
+                           0      0      0      1 0" />
```
Comment (OnkarRuikar) [inline files/en-us/web/svg/applying_svg_effects_to_html_content/index.md:167]:
```suggestion
             0      0      0      1 0" />
```

File: files/en-us/web/svg/applying_svg_effects_to_html_content/index.md [hunk 10]
```diff
@@ -156,10 +160,11 @@ You could also apply a color matrix:
...
   <filter id="f2">
-    <feColorMatrix values="0.3333 0.3333 0.3333 0 0
-                           0.3333 0.3333 0.3333 0 0
-                           0.3333 0.3333 0.3333 0 0
-                           0      0      0      1 0"/>
+    <feColorMatrix
+      values="0.3333 0.3333 0.3333 0 0
+              0.3333 0.3333 0.3333 0 0
+              0.3333 0.3333 0.3333 0 0
+             0      0      0      1 0" />
```
Comment (OnkarRuikar) [inline files/en-us/web/svg/applying_svg_effects_to_html_content/index.md:167]:
```suggestion
              0      0      0      1 0" />
```

File: files/en-us/web/progressive_web_apps/app_structure/index.md [hunk 11]
```diff
@@ -74,36 +74,53 @@ From the HTML point of view, the app shell is everything outside the content sec
...
+    <link rel="manifest" href="js13kpwa.webmanifest" />
+    <script src="data/games.js" defer></script>
+    <script src="app.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <p>
+        <a class="logo" href="https://js13kgames.com">
+        		<img src="img/js13kgames.png" alt="js13kGames" />
+        </a>
```
Comment (OnkarRuikar) [inline files/en-us/web/progressive_web_apps/app_structure/index.md:100]:
```suggestion
        <a class="logo" href="https://js13kgames.com">
          <img src="img/js13kgames.png" alt="js13kGames" />
        </a>
```

File: files/en-us/web/security/securing_your_site/turning_off_form_autocompletion/index.md [hunk 12]
```diff
@@ -28,16 +28,16 @@ You can do this either for an entire form, or for specific input elements in a f
 
 ```html
 <form method="post" action="/form" autocomplete="off">
-[…]
+  <!-- … -->
```
Comment (teoli2003) [inline files/en-us/web/security/securing_your_site/turning_off_form_autocompletion/index.md:31]:
```suggestion
  …
```

File: files/en-us/web/svg/attribute/patterntransform/index.md [hunk 13]
```diff
@@ -28,16 +28,18 @@ svg {
...
   <!-- Apply a transform on the tile -->
-  <pattern id="p1" width=".25" height=".25"
-           patternTransform="rotate(20)
+  <pattern
+    id="p1"
+    width=".25"
+    height=".25"
+    patternTransform="rotate(20)
                              skewX(30)
                              scale(1 0.5)">
```
Comment (teoli2003) [inline files/en-us/web/svg/attribute/patterntransform/index.md:37]:
```suggestion
                      skewX(30)
                      scale(1 0.5)">
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**